tls_codec/derive/tests/decode.rs

```diff
@@ -3,6 +3,8 @@ use tls_codec::{
     Deserialize, DeserializeBytes, Error, Serialize, Size, TlsSliceU16, TlsVecU8, TlsVecU16,
     TlsVecU32, VLBytes,
 };
+#[cfg(hax)]
```

Comment:
The derive crate needs the `check-cfg` for `hax` as well to avoid the warnings here.